lib/ace/mode/latex_parser.js

```diff
@@ -33,26 +62,81 @@ var LatexParser = exports.LatexParser = function(bgTokenizer) {
                 }
 
                 switch(token.type) {
-                    case "text": case "comment":
-                        var words = this.splitTextToWords(token.value, row, column);
-                        for (var i = 0, wordsLength = words.length; i < wordsLength; ++i) {
-                            if (!dictionary.check(words[i].value)) {
-                                this.myErrors.push({
-                                    row: words[i].row,
-                                    column: words[i].column,
-                                    text: "grammar error",
-                                    type: "error",
-                                    raw: words[i].value
-                                });
-                            }
-                        }
+                    case "comment":
+                        doGetDiscussions && parseRowForDiscussions.call(this, line, row);
```

Comment:
maybe parse discussions iff the line is a full line comment, i.e. `tokens.length === 1 && token.type === "comment"`?